protocols/kad/src/kbucket/key.rs

```diff
@@ -77,6 +77,11 @@ impl<T> Key<T> {
         self.bytes.distance(other)
     }
 
+    /// Exposing the hashed bytes.
+    pub fn hashed_bytes(&self) -> &GenericArray<u8, U32> {
```

Comment:
Instead of this, I think I'd prefer to expose a `[u8; 32]` on `KeyBytes` or even just a `&[u8]`. I don't want to have `GenericArray` in our public API.

Reply:
Would a byte slice be sufficient for you? Then I'd suggest we add an `as_slice` method on `KeyBytes`.

Reply:
thx for the comment. 
PR upadted to avoid using `GenericArray` in your public API.

Reply:
Thanks! What do you think of moving it to `KeyBytes` instead? And can we get away with not allocating but returning a slice?

Reply:
`KeyBytes` is not exposed in pubic API ?  

I am not sure what you mean about `can we get away with not allocating but returning a slice` ?
could you make it as an example pls? thx